src/resources/reference-data/ReferenceDataShow.tsx

```diff
@@ -44,11 +45,26 @@ export default function ReferenceDataShow({
   name
 }: PropType): React.ReactElement {
   const cName: ResourceTypes = name as ResourceTypes
-
```

Comment:
Hello @TahaKhanAbdalli - I don't like to see this array of titles twice in the code.  Could you please just declare it once?

Reply:
Sorry, are you talking about this code?
![image](https://github.com/DeepBlueCLtd/RCO/assets/50602678/e5e38696-74a5-4f09-a104-8fc7ace9111e)

Reply:
Yes - it's present in two places.